src/terraform/mlz/main.tf

```diff
@@ -513,3 +513,53 @@ module "jumpbox" {
     DeploymentName = var.deploymentname
   }
 }
+
+#####################################
+### STAGE 4: Compliance example   ###
+#####################################
+
+module "hub-policy-assignment" {
+  count = var.create_assignment ? 1 : 0
+
+  providers  = { azurerm = azurerm.hub }
+  source = "../modules/policy-assignments"
+  depends_on = [azurerm_resource_group.hub, azurerm_log_analytics_workspace.laws]
+  resource_group_name = azurerm_resource_group.hub.name
+  laws_instance_id = azurerm_log_analytics_workspace.laws.workspace_id
+  environment  = var.tf_environment # Example "usgovernment"
+  log_analytics_workspace_resource_id = azurerm_log_analytics_workspace.laws.id
```

Comment:
Can you run `terraform fmt` on this file? Should clean-up the whitespace on these assignments for us:

```plaintext
module "hub-policy-assignment" {
  count = var.create_assignment ? 1 : 0

  providers                           = { azurerm = azurerm.hub }
  source                              = "../modules/policy-assignments"
  depends_on                          = [azurerm_resource_group.hub, azurerm_log_analytics_workspace.laws]
  resource_group_name                 = azurerm_resource_group.hub.name
  laws_instance_id                    = azurerm_log_analytics_workspace.laws.workspace_id
  environment                         = var.tf_environment # Example "usgovernment"
  log_analytics_workspace_resource_id = azurerm_log_analytics_workspace.laws.id
}
```